drivers/hmis/app/models/hmis/client_alert.rb

```diff
@@ -0,0 +1,10 @@
+###
```

Comment:
Is it correct to copy this license info into the top of every file? Should I set my ide to do so automatically?

Reply:
Yes, thanks! We have a task that inserts them so we do that periodically. i don't have anything automatic with my editor. This just came up recently https://www.pivotaltracker.com/story/show/186889763/comments/239959866